erpnext/patches/v9_0/create_training_for_training_event.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2017, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	frappe.reload_doctype('Training Event')
+
+	training_events = frappe.get_all("Training Event", fields=["name", "event_name", "training", "trainer_name", \
+		"trainer_email", "supplier", "contact_number"])
+	for training_event in training_events:
+		training_event_doc = frappe.get_doc("Training Event", training_event.name)
```

Comment:
Using fields='*' you will not need this line. You have all the fields from the variable 'training_events'.